### DISCUSSION TITLE
Welcome to Beyond Better Discussions!

### DISCUSSION THREAD

**Discussion by cngarrison** [Beyond-Better/bb#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by batnom** [comment]
Howdy =) I'm an ex colleague of Charlie's testing out BB, primarily making passion project iOS apps at this stage.

I am very time and energy poor with two kids in primary school, but constantly have app or game ideas I would love to mock up.

BB has blown my mind in terms of what it can achieve from a brief prompt - how it can tie together frontend and backend and usually leave it in a compilable state is unreal!
